apps/pollbook/frontend/src/app.tsx

```diff
@@ -93,6 +100,15 @@ function AppRoot(): JSX.Element | null {
     );
   }
 
+  if (isVendorAuth(auth)) {
+    return (
+      <VendorScreen
+        logOut={() => logOutMutation.mutate()}
```

Comment:
nit: such a trivial difference but you could drop the arrow function here and below - `logOut={logOutMutation.mutate}`

Reply:
good catch. Updated in [8cf59cd](https://github.com/votingworks/vxsuite/pull/6411/commits/8cf59cd48178ad97a6a85661c3a4ff97961a1473) alongside `admin`